libkbfs/tlf_journal.go

```diff
@@ -813,6 +826,56 @@ func (e errTLFJournalNotEmpty) Error() string {
 	return "tlfJournal is not empty"
 }
 
+func (j *tlfJournal) checkServerForConflicts(ctx context.Context) error {
+	if time.Since(j.lastServerMDCheck) < tlfJournalServerMDCheckInterval ||
```

Comment:
This implicitly considers `time.Now()` rather than `config.Clock().Now()`. Not sure if that matters, but it makes it less testable.
```go
j.config.Clock().Now().Sub(j.lastServerMDCheck) < tlfJournalServerMDCheckInterval
```